include/z64actor.h

```diff
@@ -38,11 +37,45 @@ typedef struct {
     /* 0x1C */ ActorFunc draw; // Draw function
 } ActorInit; // size = 0x20
 
-typedef enum {
-    /* 0 */ ALLOCTYPE_NORMAL,
-    /* 1 */ ALLOCTYPE_ABSOLUTE,
-    /* 2 */ ALLOCTYPE_PERMANENT
-} AllocType;
+/**
+ * @see ACTOR_ALLOC_ABSOLUTE
+ */
+#define ACTOR_OVERLAY_ABSOLUTE_SPACE_SIZE 0x27A0
+
+/**
+ * The actor overlay should be allocated memory for when loading,
+ * and the memory deallocated when there is no more actor using the overlay.
+ *
+ * `ACTOR_ALLOC_` defines indicate how an actor overlay should be loaded.
+ *
+ * @note Bitwise or-ing `ACTOR_ALLOC_` types is not meaningful.
+ * The `ACTOR_ALLOC_` types are 0, 1, 2 but checked against with a bitwise and.
+ *
+ * @see ACTOR_ALLOC_ABSOLUTE
+ * @see ACTOR_ALLOC_PERSISTENT
+ * @see actor_table.h
+ */
+#define ACTOR_ALLOC_NORMAL 0
```

Comment:
Do these all render properly when generating the Doxygen? I've had them not stick to macros properly before.

Reply:
Yes?:

<details>

![image](https://user-images.githubusercontent.com/5306619/173176368-55fc04fa-f5b3-414e-9850-a879e818ed5e.png)

</details>